### PR TITLE
feat(new-study-panel): default to list view for non thumbnail series, change default fitler to all, and add more menu to thumbnail items with a dicom tag browser

### PR DESCRIPTION
**Pull request by IbrahimCSAE**
### Context

Adds several improvements to the study panel and takes it out of experimental phase.

Features:

- Default to list view for non thumbnail series
- Change default study filter to all
- Add more menu button to thumbnail items with a dicom tag browser option

### CODE REVIEW

File: platform/core/src/services/CustomizationService/CustomizationService.ts [hunk 1]
```diff
@@ -156,6 +156,12 @@ export default class CustomizationService extends PubSubService {
 
   public onModeEnter(): void {
     super.reset();
+
```
Comment (IbrahimCSAE) [inline platform/core/src/services/CustomizationService/CustomizationService.ts:159]: fix segmentation editing bug

File: platform/ui-next/src/components/ThumbnailList/ThumbnailList.tsx [hunk 2]
```diff
@@ -10,7 +10,8 @@ const ThumbnailList = ({
   onClickUntrack,
   activeDisplaySetInstanceUIDs = [],
   viewPreset,
-}) => {
+  commandsManager,
```
Comment (sedghi) [inline platform/ui-next/src/components/ThumbnailList/ThumbnailList.tsx:13]: No

Comment (sedghi) [reply platform/ui-next/src/components/ThumbnailList/ThumbnailList.tsx:13]: UI should be dummy, no services and no commands, if you need to run a command pass it as props, onWhatEverClick = {}

File: platform/ui-next/src/components/Thumbnail/Thumbnail.tsx [hunk 3]
```diff
@@ -30,8 +37,10 @@ const Thumbnail = ({
   canReject = false,
   onReject = () => {},
   isTracked = false,
+  thumbnailType = 'thumbnail',
   onClickUntrack = () => {},
-}): React.ReactNode => {
+  commandsManager,
```
Comment (sedghi) [inline platform/ui-next/src/components/Thumbnail/Thumbnail.tsx:42]: Remove please

File: platform/ui-next/src/components/StudyItem/StudyItem.tsx [hunk 4]
```diff
@@ -19,7 +19,8 @@ const StudyItem = ({
   onDoubleClickThumbnail,
   onClickUntrack,
   viewPreset = 'thumbnails',
-}) => {
+  commandsManager,
```
Comment (sedghi) [inline platform/ui-next/src/components/StudyItem/StudyItem.tsx:22]: Remove

File: platform/ui-next/src/components/StudyBrowser/StudyBrowser.tsx [hunk 5]
```diff
@@ -31,6 +31,7 @@ const StudyBrowser = ({
   servicesManager,
   showSettings,
   viewPresets,
+  commandsManager,
```
Comment (sedghi) [inline platform/ui-next/src/components/StudyBrowser/StudyBrowser.tsx:34]: remove

**Review by sedghi — COMMENTED** [review]
thanks, see my comments

**Comment by sedghi** [comment]
What is left 

- lag when large thumbnail list gets uncollapsed

I changed my mind i don't need the Tracking vs non-tracking for now

**Review by sedghi — APPROVED** [review]

**Pull request merged**